Update CSS background-color (#44336)
mdn/content: files/en-us/web/css/reference/properties/background-color/index.md

- {{cssxref("&lt;color&gt;")}}
  - : The uniform color of the background. It is rendered behind any {{cssxref("background-image")}} that is specified, although the color will still be visible through any transparency in the image.

## Description

> The `background-color` property sets the background color of an element box. The color is drawn behind any background images. By default, the background color is painted within the border box, meaning is painted behind the border, ending at the outer edge of the border-box.

Reviewer: ```suggestion
The `background-color` property sets the background color of an element box. The color is drawn behind any background images. By default, the background color is painted within the [border-box](/en-US/docs/Web/CSS/Guides/Box_model/Introduction#border_area), meaning is painted behind the border, ending at the outer edge of the border-box.
```

I would suggest linking the first usage of `border-box` to the relevant guide’s section. Also, adding a dash for consistency.

> The `background-color` property sets the background color of an element box. The color is drawn behind any background images. By default, the background color is painted within the [border-box](/en-US/docs/Web/CSS/Guides/Box_model/Introduction#border_area), meaning is painted behind the border, ending at the outer edge of the border-box.

Reviewer: ```suggestion
The `background-color` property sets the background color of an element box. The color is drawn behind any background images. By default, the background color is painted within the [border-box](/en-US/docs/Web/CSS/Guides/Box_model/Introduction#border_area), meaning it's painted behind the border, ending at the outer edge of the border-box.
```

Clipping of the `background-color` paint area is controlled via the {{cssxref("background-clip")}} property. If multiple background images are set, the background color's clipping is determined by the value of the `background-clip` associated with the bottom-most background image.

## Accessibility

It is important to ensure that the contrast ratio between the background color and the color of the text placed over it is high enough that people experiencing low vision conditions will be able to read the content of the page. A high contrast ratio also improves content accessibility for glossy-screened mobile device users when in a bright environment, such as sunlight.

Color contrast ratio is determined by comparing the luminance of the text and background color values. In order to meet current [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/), a ratio of 4.5:1 is required for text content and 3:1 for larger text such as headings. Large text is defined as 18.66px and [bold](/en-US/docs/Web/CSS/Reference/Properties/font-weight) or larger, or 24px or larger.

> - [Understanding WCAG: Perceivalble Guideline 1.4.3](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background) understanding [color contrast](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast)

Reviewer: ```suggestion
- [Understanding WCAG: Perceivable Guideline 1.4.3](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background)
- [Understanding color contrast](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast)
```

I hope I parsed it right

## Examples

### Basic example

> This example demonstrates applying `background-color` to HTML {{HTMLelement("p")}} elements using different CSS {{cssxref("color_value", "&lt;color&gt;")}} data-type values.

Reviewer: ```suggestion
This example demonstrates applying `background-color` to HTML {{HTMLelement("p")}} elements using different CSS {{cssxref("color_value", "&lt;color&gt;")}} values.
```

I would say it’s overly specific to mention data types in a demo description. Make it simpler?

#### HTML

```html
<p class="example-one">Lorem ipsum dolor sit amet, consectetuer</p>

<p class="example-two">Lorem ipsum dolor sit amet, consectetuer</p>

<p class="example-three">Lorem ipsum dolor sit amet, consectetuer</p>
```

#### CSS

> Each paragraph is set to a different background color, including explicity setting the default [`transparent`](/en-US/docs/Web/CSS/Reference/Values/named-color#transparent), an {{cssxref("rgb()")}} color function, and a {{cssxref("hex-color")}}. We also set the {{cssxref("color")}} proprty to ensure enough contrast between the text and its background.

Reviewer: ```suggestion
Each paragraph is set to a different background color, including explicitly setting the default [`transparent`](/en-US/docs/Web/CSS/Reference/Values/named-color#transparent), an {{cssxref("rgb()")}} color function, and a {{cssxref("hex-color")}}. We also set the {{cssxref("color")}} property to ensure enough contrast between the text and its background.
```
